var/spack/repos/builtin/packages/acts/package.py

```diff
@@ -219,6 +225,14 @@ class Acts(CMakePackage, CudaPackage):
         default="MAX",
         description="Log level above which examples should auto-crash",
     )
+    variant(
+        "scalar",
```

Comment:
Does the new scalar variant not need to be set on cmake configuration?

Reply:
Yes, of course :man_facepalming:. Well spotted!